Raven.Database/Queries/MoreLikeThisQueryRunner.cs

```diff
@@ -232,12 +243,32 @@ private JsonDocument[] GetJsonDocuments(MoreLikeThisQuery parameters, IndexSearc
                 .Select(hit => new JsonDocument
                 {
                     DataAsJson = Index.CreateDocumentFromFields(searcher.Doc(hit.Doc),
-                                                                new FieldsToFetch(fields, false, index.IsMapReduce ? Constants.ReduceKeyFieldName : Constants.DocumentIdFieldName)),
+                        new FieldsToFetch(fields, false, index.IsMapReduce ? Constants.ReduceKeyFieldName : Constants.DocumentIdFieldName)),
                     Etag = etag
                 })
                 .ToArray();
         }
 
+        private JsonDocument[] GetJsonDocuments(IndexSearcher searcher, Index index, string indexName, ScoreDoc[] scoreDocs)
```

Comment:
Not sure if you prefer passing a nullable value to the _original_ ```GetJsonDocuments``` method and have an ```if``` clause in there or if you think this way its better.